archetype-models/archetype-descriptor/src/main/mdo/archetype-descriptor.mdo

```diff
@@ -162,6 +162,13 @@
           <required>false</required>
           <description>Encoding to use when filtering content.</description>
         </field>
+        <field xml.attribute="true">
```

Comment:
where does xml.attribute come from? Per XML 1.0 spec "Names beginning with the string 'xml', or with any string which would match (('X'|'x') ('M'|'m') ('L'|'l')), are reserved for standardization in this or future versions of this specification."

Reply:
https://codehaus-plexus.github.io/modello/modello.html#field

Reply:
Wow, that is seriously broken. Not something we can fix in this PR, but another reason to avoid Modello.